crates/ruff_linter/src/rules/flake8_bugbear/rules/no_explicit_stacklevel.rs

```diff
@@ -11,11 +11,12 @@ use crate::checkers::ast::Checker;
 ///
 /// ## Why is this bad?
 /// The `warnings.warn` method uses a `stacklevel` of 1 by default, which
-/// limits the rendered stack trace to that of the line on which the
-/// `warn` method is called.
+/// will output a stack frame of the line on which the "warn" method
+/// is called. Setting it to a higher number will output a stack frame
+/// from higher up the stack.
 ///
-/// It's recommended to use a `stacklevel` of 2 or higher, give the caller
-/// more context about the warning.
+/// It's recommended to use a `stacklevel` of 2 or higher, to give the caller
+/// a more appropriate context about the warning.
```

Comment:
Actually I think this line with just "more context about the warning" sounds better as is. (But the "give"--> "to give" looks good)

Reply:
Since it was a small nit I just applied it for you 😄 - thank you for the contribution!